lib/rules.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Rules do
+  def winning_combos(board) do
+    Enum.into(Board.row(board), Board.column(board))
+    |> Enum.into(Board.diagonal(board))
+  end
+
+  def game_in_progress?(board) do
+    (win?(board) == false && draw?(board) == false)
+    || (win?(board) == nil && draw?(board) == false)
```

Comment:
Why would `win?` ever return nil?

Reply:
because of the function I used to determine if there is a win. 
```
  def win?(board) do
    if board == Board.empty_board() do
      false
    else
      Enum.find_value(winning_combos(board), fn(x) -> identical_player_symbol?(x) end)
    end
  end```

`find_value` returns nil if it doesn't find anything.